### DISCUSSION TITLE
I had a memory of Nature Renderer operating via a terrain material in the past. Currently it uses a script. Did it ever use a Terrain Material

### DISCUSSION THREAD

**Discussion by LithuanianDude** [visualdesigncafe/nature-renderer#29] [Help]
I started a new project and imported Nature Renderer after not using it for a long time. I found out that it uses a script attached to the terrain to work, In my current project, I can't use custom scripts for the current project (modding limitation).
But I have this weird memory that it used to use a terrain material to work instead of a script. Was this the case?

**Comment by mroshaw** [comment]
I can't imagine how that would work. I don't think the terrain material has anything to do with tree and detail rendering. Are you thinking of something like Microsplat, perhaps?

**Reply by LithuanianDude** [reply]
I may have been. Thanks for the answer!
